src/components/Main/Main.js

```diff
@@ -22,24 +21,98 @@ function tellColorName(house) {
   return color;
 }
 
-function Main({ characterName, house, imgURL, ancestry }) {
-  const color = tellColorName(house);
+function Main(props) {
```

Comment:
`Main` is not a good name for this component. It is not a `main` HTML element, which it indicates, but a card or a component which shows details of a character. Please rename it to `Card`